scripts/globals/mobskills/pollen.lua

```diff
@@ -17,13 +17,24 @@ end
 
 mobskill_object.onMobWeaponSkill = function(target, mob, skill)
     local potency = skill:getParam()
+    local id = mob:getID()
 
     if (potency == 0) then
         potency = 12
     end
 
     potency = potency - math.random(0, potency/4)
 
+    if -- KSNM Infernal Swarm
+        id == 17621248 or -- IDs for Beelzebub
+        id == 17621258 or
+        id == 17621268
+    then
+        print("found ID")
+        potency = 55
```

Comment:
Got any sources for this?

Reply:
proof is linked in the commit. Should I also link in the PR comment?

Reply:
A single forum post from 2005 doesn't do it for me I'm afraid. Hopefully, this is covered in a cap. I'm 70% sure that this KSNM is now uncapped on retail, so it wouldn't be hard to hop on and find out

Reply:
https://ffxiclopedia.fandom.com/wiki/Infernal_Swarm#:~:text=All%20bees%20have%20Pollen%20and%20Sharp%20Sting.%20Beelzebub%27s%20Pollen%20can%20heal%20for%20a%20couple%20thousand%20HP. is another source I originally linked in the commit

Reply:
After reviewing the live captures you linked me to, 2000 was definitely too powerful. Seemed to hover around ~700-~900. Fixed was pushed.